src/modules/Laboratorio/Editar/index.js

```diff
@@ -1,19 +1,55 @@
-import React from 'react';
-
+import React, {useState, useEffect} from 'react';
 import Menu from '../../../utils/components/Menu';
 import Title from '../../../utils/components/Title';
 import Toolbar from '../../../utils/components/Toolbar';
 import Formulario from './components/Formulario';
+import {Redirect} from 'react-router-dom';
+import api from '../../../services/api';
+
+
+const Editar =  (props) => {
+  const [shouldRedirect,setShouldRedirect] = useState(false);
+  const [location, getLocation] = useState([]);
```

Comment:
Normalmente a comunidade usa o padrão `[something, setSomething]` com o hook `useState()`. Acho que seria uma boa utilizar aqui também

```suggestion
  const [location, setLocation] = useState([]);
```